gems/aws-sdk-core/lib/aws-sdk-core/refreshing_credentials.rb

```diff
@@ -36,12 +36,6 @@ def credentials
       @credentials
     end
 
-    # @return [Time,nil]
-    def expiration
-      refresh_if_near_expiration!
```

Comment:
I'm a little worried about changing this behavior - I suspect we might rely on that in some places?

Reply:
I don't see any cases in the SDK but certainly can in the wild. My concern is that by fetching expiration (after credentials) they will get refreshed.